### PR TITLE
Create Reconstruction Page

### PR DESCRIPTION
**Pull request by danny237**

### CODE REVIEW

**Comment by silwalanish** [comment]
@danny237 Change the title to something meaningful.

**Comment by danny237** [comment]
> @danny237 Change the title to something meaningful.

ok, I will discuss with kishor.

File: src/Pages/Reconstruction.jsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+
+import { authenticate } from '../Middlewares/authenticate';
+import { asPage } from '../Middlewares/asPage';
+
+import '../Themes/Reconstruction.css';
```
Comment (KGhising) [inline src/Pages/Reconstruction.jsx:6]: How about importing css in app.scss file?

**Review by silwalanish — APPROVED** [review]

**Review by SG-coder — APPROVED** [review]

**Review by KGhising — APPROVED** [review]

**Comment by silwalanish** [comment]
@danny237 Conficts. Merge master to your branch.

File: src/Pages/Reconstruction.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (danny237) [reply src/Pages/Reconstruction.jsx:6]: ok, I will do !

**Pull request merged**